pom.xml

```diff
@@ -2,9 +2,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <properties>
-        <awssdk.version>2.19.2</awssdk.version>
-        <aws-java-sdk.version>1.12.370</aws-java-sdk.version>
-        <kcl.version>2.5.2</kcl.version>
+        <awssdk.version>2.25.11</awssdk.version>
+        <aws-java-sdk.version>1.12.696</aws-java-sdk.version>
+        <kcl.version>2.5.8</kcl.version>
```

Comment:
Still need the -SNAPSHOT version for testing. We should not be using this version as it will confuse local builds with public releases